### ISSUE TITLE
Publishing on open-vsx

### ISSUE THREAD

**Issue by GoodingJamie** [AlbertoPdRF/root-file-viewer#39]
Hi @AlbertoPdRF, thanks a lot for the really nice tool! I've moved recently from VSCode to another VSCode-based IDE and found that non-Microsoft VSCode-based IDEs aren't permitted to use Microsoft's extension marketplace. Instead, open source marketplaces are used, e.g., open-vsx as a common alternative. The instructions to publish to this (posted below) seem fairly comprehensive and once you have an existing published package then publishing to open-vsx seems (at first glance) to be relatively straightforward. Completely optional of course but though I'd through the suggestion out there :D

_open-vsx publishing instructions:_ https://github.com/EclipseFdn/open-vsx.org/wiki/Publishing-Extensions

**Comment by MohamedElashri** [comment]
I second this as now many people are using VS Code forks that cannot use the official VS Code extension marketplace.

**Comment by fjonasALICE** [comment]
just on a sidenote, if you still want to install this extension in a VS Code fork even though this is not published in open-vsx:

1. Go to the release page of this repo and download the .vsix file
2. In VS Code fork (for me cursor) Ctrl + Shift + P
3.  Extensions: Install from VSIX ...

I don't know why it took me so long to figure out that you can still manually install extensions from vsix files ...

**Comment by MohamedElashri** [comment]
> just on a sidenote, if you still want to install this extension in a VS Code fork even though this is not published in open-vsx:
> 
>     1. Go to the release page of this repo and download the .vsix file
> 
>     2. In VS Code fork (for me cursor) Ctrl + Shift + P
> 
>     3. Extensions: Install from VSIX ...
> 
> 
> I don't know why it took me so long to figure out that you can still manually install extensions from vsix files ...

And you need to do this to update the extension each time there is an update.

**Comment by fjonasALICE** [comment]
yes agreed it is not ideal. I just wanted to put it here in case someone stumbled over it. I also advocated for putting this on open-vsx (created an issue before seeing this issue here) but it seems the author is not responsive or does not want this for some reason.
